indico/modules/events/papers/templates/management/competences.html

```diff
@@ -0,0 +1,34 @@
+{% from 'forms/_form.html' import form_header, form_field, form_footer %}
+{% extends 'layout/conference_page_base.html' %}
+
+{% block description %}
+    {%- trans -%}
+        Defining competences to the members of the teams may be helpful for the managers and judges during the paper assignment process.
+    {%- endtrans -%}
+{%- endblock %}
+
+{% block content -%}
+    <div id="competences-dialog-content">
+        {{ form_header(form) }}
+            <table class="i-table-widget">
```

Comment:
Well, `{{ form_header(form) }}` will technically open an HTML tag, which will close by calling the `form_footer`.